common/templates/base_templates/base.html

```diff
@@ -45,5 +57,8 @@
       {% block body %}{% endblock %}
     </div>
     {# END_FEATURE bootstrap_messages #}
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
```

Comment:
Bootstrap 5 no longer requires jquery, and the bootstrap.bundle.min.js already includes popper, so I think it's sufficient to remove both jquery and popper.